apps/emqx_machine/src/emqx_machine_boot.erl

```diff
@@ -47,7 +47,7 @@ post_boot() ->
     ok = ensure_apps_started(),
     ok = print_vsn(),
     ok = start_autocluster(),
-    ?AUDIT(alert, "from_cli", #{time => logger:timestamp(), event => "emqx_start"}),
+    ?AUDIT(alert, cli, #{time => logger:timestamp(), event => "emqx_start"}),
```

Comment:
What is the informal "schema" of the audit messages? Here the event "kind" is placed as `event` field in the meta map, but [here](https://github.com/emqx/emqx/pull/11695/files#diff-b949fddc8ef716937f819cd9007be07dd9dd23bc9140d14bc03486a30b207cc5R70) it's in the message field...

Reply:
Honestly, there really isn't a definitive schema :( but now I have at least made stop/start look consistent.